apps/shell/Kconfig

```diff
@@ -31,7 +32,8 @@ config TASH_COMMAND_INTERFACE
 
 config TASH_CMDTASK_STACKSIZE
 	int "TASH task stack size to run command with ASYNC type"
-	default 4096
+	default 4096 if !TIZENRT_LITTLE_PROFILE
+	default 2048 if TIZENRT_LITTLE_PROFILE
```

Comment:
At above your modification, if ```TIZENRT_LITTLE_PROFILE``` is enabled, shell is not come up. Why is this needed?